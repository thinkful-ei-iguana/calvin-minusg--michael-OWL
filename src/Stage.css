.stage {
  flex-wrap: wrap;
  background-color: #1e2835;
  color: #4c6fa0;
  margin: auto;
  padding-left: 200px;
  padding-right: 200px;
  padding-bottom: 75px;
  padding-top: 50px;
  width: 60%;
  float: right;
}

.avatar-on-stage {
  width: 50px;
  align-content: center;
  clear: both;
}

.userContainer {
  width: 50px;
  color: white;
  clear: both;
}
