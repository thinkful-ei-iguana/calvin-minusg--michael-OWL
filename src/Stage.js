import React from "react";
import "./Stage.css";

export default function stage(props) {
  let newProps = props.participants.filter(
    participant => participant.onStage === true
  );
  return (
    <div className="stage">
      {newProps.map(participant => (
        <section key={participant.id} className="userContainer">
          <div>
            <h6>{participant.name}</h6>
          </div>
          <div>
            <img alt={participant.name} src={participant.avatar} />
          </div>
        </section>
      ))}
    </div>
  );
}
