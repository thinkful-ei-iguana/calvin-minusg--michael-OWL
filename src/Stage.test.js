import React from "react";
import Stage from "./Stage";
import ReactDOM from "react-dom";
import STORE from "./STORE";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Stage participants={STORE} />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("renders the UI as expected", () => {
  const tree = renderer.create(<Stage participants={STORE} />).toJSON();
  expect(tree).toMatchSnapshot();
});
