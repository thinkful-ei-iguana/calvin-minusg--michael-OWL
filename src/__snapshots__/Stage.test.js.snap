// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="stage"
>
  <section>
    <div>
      <h6>
        Koren Templeton
      </h6>
    </div>
    <div>
      <img
        alt="Koren Templeton"
        src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      />
    </div>
  </section>
  <section>
    <div>
      <h6>
        Ashla Attwool
      </h6>
    </div>
    <div>
      <img
        alt="Ashla Attwool"
        src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
      />
    </div>
  </section>
</div>
`;
